tests/publiccloud/cloud_netconfig.pm

```diff
@@ -0,0 +1,81 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: cloud-netconfig-{azure, gc2, gce}
+# Summary: This test ensures the consistency of cloud-netconfig's
+# functionality. The test shall be conducted on a VM in the cloud
+# infrastructure of a supported CSP.
+#
+# This test only contains minimal functionality that needs to be
+# extended in future.
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use Test::Assert 'assert_equals';
+use testapi;
+use version_utils 'is_sle';
+
+sub run {
+    my ($self, $args) = @_;
+    my $instance = $args->{my_instance};
+    my $os_version = get_var('VERSION');
+
+    my $cloud_netconfig_svc_status = $instance->ssh_script_run(
+        cmd => 'systemctl status cloud-netconfig.service');
+    if ($cloud_netconfig_svc_status == 0 || $cloud_netconfig_svc_status == 3)
+    {
+        record_info('Success', 'Status of cloud-netconfig.service is OK.');
+    }
+    else {
+        die('cloud-netconfig.service status appears to be incorrect!');
+    }
+    $instance->ssh_assert_script_run(
+        cmd => 'systemctl status cloud-netconfig.timer',
+        fail_message => 'cloud-netconfig.timer appears not to be started.'
+    );
+
+    # 75-persistent-net-generator.rules is required for SLES 12-SPX
+    if (is_sle('<=12-sp5')) {
+        $instance->ssh_assert_script_run(
+            cmd => 'test -s /etc/udev/rules.d/75-persistent-net-generator.rules',
```

Comment:
How about using a variable for these long paths? :)

Reply:
Thanks for the suggestion. I have updated the PR. New VR incoming.

Reply:
https://openqa.suse.de/tests/12897652